### ISSUE TITLE
Specify a model for deprecating vocabularies

### ISSUE THREAD

**Issue by baskaufs** [tdwg/vocab#17]
The [TDWG Ontologies](https://github.com/tdwg/ontology) provides examples of vocabularies that are no longer under development or that are no longer recommended for use.  It occurs to me that the maintenance spec should describe a process for deprecating such vocabularies and the documentation standard should specify a graceful way of indicating the deprecation in the RDF without breaking existing applications (i.e. URIs should still dereference, but the metadata should allow a semantic client to learn of the deprecation).  There is some suggested language for this in Section 2.6.1 of the [VoMaG report](https://github.com/tdwg/vocab/blob/master/gbif_TDWG_Vocabulary_Management_Task_Group_en_v1.0.pdf) regarding the fate of the TDWG Ontologies.  I'm going to implement these suggestions on ontology documents and try to get some feedback from the RDF/OWL Task Group about whether they follow best practices.  See the [TDWG Ontologies issues tracker](https://github.com/tdwg/ontology/issues) for what's going on.

**Comment by baskaufs** [comment]
I have finished marking up the TDWG Ontology documents to indicate their deprecation.  [Here is the diff](https://github.com/tdwg/ontology/commit/269406d539aca780656fcdeee841860d12026dca?diff=split) showing what I did. Points:
1. I updated dcterms:modified and incremented the version number so it would be possible to know that the deprecated version is more recent.
2. I used an owl:deprecated property for the ontology itself to indicate to semantic clients that the ontology was deprecated.
3. I used an rdfs:comment property to indicate to humans that the ontology was no longer under development and to suggest a place to look for a replacement.
4. I used an rdfs:seeAlso property to let a semantic client follow its nose to an rdf document that might contain information about the replacement vocabulary.  The VoMaG Report recommended including this information within the ontology document itself, but since for many ontologies I'm not sure what the recommended replacement should be, I put it in a separate document that could be managed and updated more easily.  That document is https://raw.githubusercontent.com/tdwg/ontology/master/replacements.rdf and uses dcterms:isReplacedBy as a property of the ontology to indicate the replacement vocabulary.  Right now it's a stub, but hopefully in the future suggestions can be included for all of the vocabularies.

This is the approach that made sense to me and it may serve as a springboard for discussion of a formal recommendation.

The situation was more complicated for deprecating particular .owl documents that duplicated information in other .rdf documents.  See [the diff here](https://github.com/tdwg/ontology/commit/8d7e72fcd215ba410ba3d0ba21ec23d1d01e6857).  In this case, I did not want to actually deprecate the ontology, just the document.  So I created a description of the .owl document, linked it to the ontology using dcterms:references, then deprecated the .owl document itself.  In this case, I specifically indicated in the .owl document itself using dcterms:isReplacedBy that the .rdf version of the document was a replacement for the .owl document.
